modules/ROOT/pages/provisioning-openstack.adoc

```diff
@@ -0,0 +1,97 @@
+= Provisioning Fedora CoreOS on OpenStack
+
+This guide shows how to provision new Fedora CoreOS (FCOS) nodes on an
+OpenStack cloud environment, either private, or public (like https://vexxhost.com/[VEXXHOST]).
+
+== Prerequisites
+
+Before provisioning a FCOS machine, you must have an Ignition configuration file
+containing your customizations. If you do not have one, see
+xref:producing-ign.adoc[Producing an Ignition File].
+
+You also need to have access to an OpenStack environment and a functioning
+https://docs.openstack.org/python-designateclient/latest/user/shell-v2.htm[`openstack` CLI]. 
+Typically you'll https://docs.openstack.org/python-openstackclient/latest/configuration/index.html[configure the client]
+by using a `clouds.yaml` file or via environment variables. If you're starting from scratch, this
+environment may need networks, ssh keypairs, security groups, etc.. set up. Please consult the
+https://docs.openstack.org/[OpenStack Documentation] to learn more.
+
```

Comment:
Should we mention here what version of OpenStack the instructions below are tested against?

Reply:
Added to new upload.